src/pages/test-report/components/competency/competency.ts

```diff
@@ -126,6 +126,43 @@ export class CompetencyComponent {
 
   getLabel = (): string => competencyLabels[this.competency];
 
+  recordSerious = (): void => {
+    this.store$.dispatch(new AddSeriousFault(this.competency));
```

Comment:
are recordSerious and recordDangerous used?

Reply:
Good point, it was just experimental code. Removed them now.